ale_linters/markdown/mdl.vim

```diff
@@ -0,0 +1,41 @@
+" Author: Steve Dignam <steve@dignam.xyz>
+" Description: Support for mdl, a markdown linter
+
+if exists('g:loaded_ale_linters_markdown_mdl')
+    finish
+endif
+
+let g:loaded_ale_linters_markdown_mdl = 1
+
+function! ale_linters#markdown#mdl#Handle(buffer, lines) abort
+    " matches: '(stdin):173: MD004 Unordered list style'
+    let l:pattern = ':\(\d*\): \(.*\)$'
+    let l:output = []
+
+    for l:line in a:lines
+        let l:match = matchlist(l:line, l:pattern)
+
+        if len(l:match) == 0
+            continue
+        endif
+
+        call add(l:output, {
+        \   'bufnr': a:buffer,
+        \   'lnum': l:match[1] + 0,
+        \   'vcol': 0,
+        \   'col': -1,
```

Comment:
I just noticed that the column number here is -1. Could you set that to 0 or 1 instead? I don't think the plugin expect negative column numbers anywhere.